cookiecutter/{{ cookiecutter.package_name }}/{{ cookiecutter.import_name }}/__init__.py

```diff
@@ -0,0 +1,106 @@
+        # does not fit this bill, so please choose something better for your
+        # own component :)
+        "{{ cookiecutter.import_name }}",
+        # Pass `url` here to tell Streamlit that the component will be served
+        # by the local dev server that you run via `npm run start`.
+        # (This is useful while your component is in development.)
+        url="http://localhost:3001",
+    )
+else:
+    # When we're distributing a production version of the component, we'll
+    # replace the `url` param with `path`, and point it to to the component's
+    # build directory:
+    parent_dir = os.path.dirname(os.path.abspath(__file__))
+    build_dir = os.path.join(parent_dir, "frontend-react/build")
+    _component_func = components.declare_component("{{ cookiecutter.import_name }}", path=build_dir)
+
+
+# Create a wrapper function for the component. This is an optional
+# best practice - we could simply expose the component function returned by
+# `declare_component` and call it done. The wrapper allows us to customize
+# our component's API: we can pre-process its input args, post-process its
+# output value, and add a docstring for users.
+def {{ cookiecutter.import_name }}(name, key=None):
+    """Create a new instance of "my_component".
+
+    Parameters
+    ----------
+    name: str
+        The name of the thing we're saying hello to. The component will display
+        the text "Hello, {name}!"
+    key: str or None
+        An optional key that uniquely identifies this component. If this is
+        None, and the component's arguments are changed, the component will
+        be re-mounted in the Streamlit frontend and lose its current state.
+
+    Returns
+    -------
+    int
+        The number of times the component's "Click Me" button has been clicked.
+        (This is the value passed to `Streamlit.setComponentValue` on the
+        frontend.)
+
+    """
+    # Call through to our private component function. Arguments we pass here
+    # will be sent to the frontend, where they'll be available in an "args"
+    # dictionary.
+    #
+    # "default" is a special argument that specifies the initial return
+    # value of the component before the user has interacted with it.
+    component_value = _component_func(name=name, key=key, default=0)
+
+    # We could modify the value returned from the component if we wanted.
+    # There's no need to do this in our simple example - but it's an option.
+    return component_value
+
+
+# Add some test code to play with the component while it's in development.
+# During development, we can run this just as we would any other Streamlit
+# app: `$ streamlit run my_component/__init__.py`
+if not _RELEASE:
+    import streamlit as st
+
+    st.subheader("Component with constant args")
+
+    # Create an instance of our component with a constant `name` arg, and
+    # print its output value.
+    num_clicks = my_component("World")
```

Comment:
We should have proper component name invocation here, should not this be replaced with `{{ cookiecutter.import_name }}`?

Reply:
Fixed.